module/project/templateTasks/helper.js

```diff
@@ -649,7 +649,8 @@ module.exports = class ProjectTemplateTasksHelper {
                     tasks = await this.taskDocuments({
                         _id : {
                             $in : templateDocument[0].tasks
-                        }
+                        },
+                        parentId : { $exists : false }
```

Comment:
@aman-tunerlabs - Is this change required ?

Reply:
Yeah required . When tested it was creating child tasks in tasks array but should be only in children of a tasks.